src/wmnds/patterns/content-cards/email-sign-up/_email-sign-up.njk

```diff
@@ -0,0 +1,64 @@
+{% macro wmndsContentCardEmailSignUp(params) %}
+{# imports #}
+{% from "wmnds/components/content-card/_content-card.njk" import wmndsContentCard %}
+{% from "wmnds/components/form-elements/input/_input.njk" import wmndsInput %}
+{% from "wmnds/components/button/_button.njk" import wmnButton %}
+{% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
+{# Set #}
+{% set formClass = " wmnds-is--open" if params.expanded else "" %}
+{# Button variables #}
+{% set btnText = "Sign up for updates" if params.expanded or params.externalLink else "Continue" %}
+{% set btnClass = " wmnds-btn--primary" if not params.externalLink else ""  %}
+{% set btnSpacingClass = " wmnds-m-t-xsm" if not params.expanded else ""  %}
+{% set btnType = "type=\"submit\"" if params.expanded or params.externalLink else "" %}
+{# Create a string where we can use the nunjucks '| replace' #}
+{% set inputId = "esu-form-%%" + ( "-" + params.exampleSuffix or "") %}
+
+{% call wmndsContentCard() %}
+    <!-- .wmnds-col-* is just used to set the width in the example -->
+    <div class="wmnds-p-sm wmnds-col-10-24">
```

Comment:
Change to wmnds-col-1 so the component is flexible as wmnds-col-10-24 doesn't display well on mobile.
```suggestion
    <div class="wmnds-p-sm wmnds-col-1">
```